src/components/NavBar/NavBar.jsx

```diff
@@ -103,6 +105,36 @@ function NavBar() {
                             <Link to="/" onClick={handleLogout}>
                                 Log Out
                             </Link>
+                            
```

Comment:
This doesn't immediately update - if I log in as a non staff, then log in as staff I don't see these in the nav bar until I refresh. Same as if I've been logged in as staff then log in as normal user, I see these in the nav bar until I refresh

Reply:
Yeah I had the same - I suspect there's another bit of logic I need to add somewhere.

Reply:
Fixed by:
Set redirect in post-login to profile page
Use the profile page 'get self' to check if is_staff=true
If so, update auth (setAuth)
Point the nav bar check at this state's property